### DISCUSSION TITLE
Ability to run multiple e2e test with different flavors

### DISCUSSION THREAD

**Discussion by Biplab-Dutta** [leancodepl/patrol#2133] [Q&A]
Let's consider I have two test files.
- file1_test.dart
- file2_test.dart

Is it possible to make configurations such that I get to run `file1_test.dart` with dev flavor and the other test with qa flavor? I don't want to keep running each test separately. I was looking for just one single command that i can use to run both these tests with different flavors.

**Comment by AnthonyDadeWT** [comment]
I think that this is just a limitation of using flavors, because they are determined at build-time, instead of being able to change your environment during run-time. It's easier said than done, but if you had a debug menu that could programmatically switch your environments while the app is running, then you'd be able to achieve this by switching environments during an integration test. Sadly, I don't think its currently possible to do what your asking for in this post using flavors.

**Reply by Biplab-Dutta** [reply]
Aah alright. Thanks for the response.

**Reply by AnthonyDadeWT** [reply]
This is just my thoughts, someone else may have another solution though? I would keep this open since I'm interested as well.
